code-review-tester-service/Program.cs

```diff
@@ -3,4 +3,10 @@
 
 app.MapGet("/", () => "I'm ALIVE!");
 
+app.MapGet("/tests/", (string? level) => 
+{
+    string[] availableLevels = ["Easy", "Medium", "Hard", "Extra Hard"];
+    return availableLevels;
```

Comment:
This should return a single string indicating the level the user requested.

Maybe it would be more obvious if the return was "Test level: [input]"